helix-term/src/commands.rs

```diff
@@ -1052,6 +1053,36 @@ fn goto_file_impl(cx: &mut Context, action: Action) {
     }
 }
 
+fn goto_url(cx: &mut Context) {
+    use helix_lsp::Url;
+
+    let (view, doc) = current_ref!(cx.editor);
+    let selection = doc.selection(view.id).primary();
+    let text = doc.text();
+
+    let text = if selection.to() - selection.from() == 1 {
+        let current_word = movement::move_next_long_word_start(
+            text.slice(..),
+            movement::move_prev_long_word_start(text.slice(..), selection, 1),
+            1,
+        );
+        text.slice(current_word.from()..current_word.to())
+            .to_string()
+    } else {
+        text.slice(selection.from()..selection.to()).to_string()
+    };
+
+    match Url::parse(&text) {
```

Comment:
Is there a reason to parse a URL? What happens if you run this command with some arbitrary text?